src/main/resources/plugin.yml

```diff
@@ -26,3 +26,8 @@ permissions:
         default: op
         children:
             townyresources.admin.command.reload: true
+            townyresources.admin.command.bypass: true
+
+    townyresources.bypass:
+        description: User always bypasses the extraction limits
+        default: false

```

Comment:
I'm wondering if this should default: op.  Thoughts?

Reply:
I would have to say definitely not, I'm OP on my own server yet I want to play on it and have the same rights as every other player.

Reply:
Sounds good